google-cloud-spanner-cassandra/src/main/java/com/google/cloud/spanner/adapter/Adapter.java

```diff
@@ -62,12 +65,19 @@ final class Adapter {
    * @param port The local TCP port number that the adapter server should listen on.
    * @param numGrpcChannels The number of gRPC channels to use for communication with the backend
    *     Spanner service.
+   * @param maxCommitDelay The max commit delay to set in requests to optimize write throughput.
    */
-  Adapter(String databaseUri, InetAddress inetAddress, int port, int numGrpcChannels) {
+  Adapter(
+      String databaseUri,
+      InetAddress inetAddress,
+      int port,
+      int numGrpcChannels,
+      Optional<Duration> maxCommitDelay) {
```

Comment:
Would it be possible to create an `Options` interface to accommodate future fields without needing to pass them individually? See Java-spanner ref: https://github.com/googleapis/java-spanner/blob/5f924b7252dc851c7c9ab21fe044e7d9d273f1fc/google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java#L953